apps/fluent-tester/src/RNTester/TestComponents/PersonaCoin/CustomizeUsage.tsx

```diff
@@ -74,6 +82,15 @@ export const CustomizeUsage: React.FunctionComponent<{}> = () => {
           }}
         />
 
+        <TextInput
+          style={[commonStyles.textBox, textBoxBorderStyle]}
+          placeholder="Icon stroke color"
+          blurOnSubmit={true}
+          onSubmitEditing={(e) => {
+            setIconStrokeColor(e.nativeEvent.text);
```

Comment:
if the text is empty, you may want to set it as `undefined` to verify that setting `undefined` stroke color is working.

Reply:
I changed the default of the start var to `undefined`.